### DISCUSSION TITLE
PS3 hard locks trying to mount /net0/PKG

### DISCUSSION THREAD

**Discussion by tlrmcknz** [aldostools/webMAN-MOD#1036] [Q&A]
1.47.45 on Evilnat 4.91 CEX, ps3netsrv 20240210a.

Didn't want to open a bug because I haven't used this cfw or ps3netsrv before, so it could be user error. I'm trying to mount /net0/PKG to /dev/bdvd and the console locks up instantly. Tried two different ps3netsrv servers, one linux using the docker container, one windows. Same results.

Logs from windows server:

Waiting for client...
[0] Connection from 192.168.1.197
building virtual iso...
open /PKG

Then just instant lockup on the PS3. Anyone else experiencing this?

**Comment by aldostools** [comment]
It seems that there is a bug in Evilnat 4.91 that freezes when a PKG is tried to be installed.

You can downgrade to 4.90 and install from there.

**Comment by aldostools** [comment]
I have been doing more tests and seems that ps3netsrv is not working on 4.91. I just reported it to Evilnat

**Reply by tlrmcknz** [reply]
Right on, appreciate you looking into it.
